pyproject.toml

```diff
@@ -227,8 +228,10 @@ line-length = 100
 
 
 [tool.coverage]
-
     [tool.coverage.run]
+        parallel = true
+        concurrency = ["thread", "multiprocessing"]
+        source = ["srttools"]
```

Comment:
Without this line somehow processes spawned by multiprocessing.Process could not be tracked. So weird.